client/src/components/_common/Searchbar/Searchbar.jsx

```diff
@@ -26,19 +27,15 @@ const Searchbar = ({
     location.search
   );
 
-  const displayName = useSystemDisplayName({ system, scheme });
-  let sectionName;
 
-  if (siteSearch) {
-    sectionName = 'Site';
-  } else if (scheme === 'projects') {
-    sectionName = 'Workspace';
-  } else if (scheme === 'jobs') {
-    sectionName = 'Jobs';
-  } else {
+  let displayName;
+  if (!sectionName) {
+    displayName = useSystemDisplayName({ system, scheme });
     sectionName = displayName;
```

Comment:
Would it be simpler to remove `displayName` altogether?
```
  if (!sectionName) {
    sectionName = useSystemDisplayName({ system, scheme });
```

Reply:
Yup that makes sense. Thanks for the catch!